### PR TITLE
Fevm update

### PR DESCRIPTION
**Pull request by shyam-patel-kira**

### CODE REVIEW

**Review by alessandromaci — COMMENTED** [review]
Following a call with @HabacucMX, I understood that we need to retrieve a CID after the storage deal is completed. This CID will then be pushed to IPFS. How can we get this CID via the contract functions?

File: contracts/StorageDeal.sol [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    function createDeal(
+        address minerAddress,
+        address clientAddress,
+        uint256 startDate,
+        uint256 endDate,
+        uint256 pricePerByte,
+        uint256 dataSize,
+        bytes32 fileHash
+    ) public {
+        bytes32 dealId = sha256(
```
Comment (alessandromaci) [inline contracts/StorageDeal.sol:32]: how do we know the `dealId` in case I want to use the `queryDeal` or `updateDeal`?

Comment (HabacucMX) [reply contracts/StorageDeal.sol:32]: I think we will need to use js IPFS to get the CID from the deal id

https://js.ipfs.tech/

**Comment by alessandromaci** [comment]
I will merge it for now

**Pull request merged**